.circleci/deploy-to-pantheon.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
> It seems like this script is the same as build:env:create

@greg-1-anderson the difference here is we are only creating environments for pull requests. If `build:env:create` has a `--pr-only` flag or similar we could probably reduce/remove this script